cpp/ql/src/semmle/code/cpp/exprs/ArithmeticOperation.qll

```diff
@@ -3,9 +3,9 @@ import semmle.code.cpp.exprs.Expr
 /**
  * A C/C++ unary arithmetic operation.
  *
- * This is an abstract base QL class.
+ * This is an base QL class.
```

Comment:
You can simply delete this line (and the blank line above it) now.

Reply:
Good point. I will do that!

Reply:
This has not been addressed.

Reply:
My bad! I have removed it now.